libnd4j/tests_cpu/layers_tests/DeclarableOpsTests11.cpp

```diff
@@ -477,6 +477,629 @@ TEST_F(DeclarableOpsTests11, log_loss_grad_test13) {
     delete results;
 }
 
+TEST_F(DeclarableOpsTests11, ImageResizeBicubic_Test1) {
+
+    NDArray input    = NDArrayFactory::create<float>('c', {1, 7, 7, 1}, {
+             1, 2.1, 3.15, 4.2, 5.15,  6.1,   7,
+             8, 9.1,  10.,  11, 12.9, 13.1,  14,
+            15, 16.,  17.,  18,   19,  20.,  21,
+            22, 23.,  24.,  25,   26,   27,  28,
+            30,  31,   32,  33,  34.,   35,  36,
+            37,  38,   39,  40,  41.,   42,  43,
+            44,  45,   46,  47,  48.,   49,  50
+    });
+    //NDArray<float> paddings('c', {3,2}, {0,0, 0,1, 0,0});
+    //NDArray<float> expected('c', {2,4,4}, {1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,0.,0.,0.,0.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,0.,0.,0.,0.});
+    NDArray expected = NDArrayFactory::create<float>('c', {1, 30, 30, 1}, {
+        1.         ,1.1976162  ,1.4174359  ,1.6775769  ,1.9961575  ,2.3283265 ,
```

Comment:
If your array is of float type, please make sure your initial values are floats too. Otherwise some compilers will complain.